setup.py

```diff
@@ -15,21 +15,38 @@
 permissions and limitations under the License.
 
 This product includes software developed at data.world, Inc.(http://www.data.world/).
-'''
+"""
+
 from setuptools import setup
+from os import path
+import re
+
+def read(*paths):
+    filename = path.join(path.abspath(path.dirname(__file__)), *paths)
+    with open(filename) as f:
+        return f.read()
+
+def find_version(*paths):
+    contents = read(*paths)
+    match = re.search(r'^__version__ = [\'"]([^\'"]+)[\'"]', contents, re.M)
+    if not match:
+        raise RuntimeError('Unable to find version string.')
+    return match.group(1)
```

Comment:
this is all fine - sure feels like these util methods should be in a library somewhere instead of cut/pasted into every `setup.py`, but `<shrug/>`